python/packages/ai/scripts/lint.py

```diff
@@ -4,7 +4,13 @@
 """
 
 import subprocess
+import sys
 
 
 def lint():
-    subprocess.run(["poetry", "run", "pylint", "teams", "scripts", "tests"], check=True)
+    result = subprocess.run(["poetry", "run", "pylint", "teams", "scripts", "tests"], check=False)
```

Comment:
to keep the linting strict is better I think. If you have suggestions about certain rules you think shouldn't be enabled we can discuss that but to me any warning/error of lint should fail the build since thats the reason we have it there to begin with.

Reply:
I agree we need to do our best to try to fix the lint issues. But since we're in early stage of the project, there's no need to has such strict rules. The change does not fail lint if there's only warning. Errors will always fail the lint like before.

Currently there's only warnings about TODO comments, which is acceptable considering we're in early stage of the project. And TODO comments should be better than other item tracking system because it's not easy to lose track - we can always see them in our code.

And disabling a rule may not always be the best choice. Like the TODO comment rule, we still want to see them. But it's not high priority to fix them right now. I personally prefer to not disable any lint rule globally unless there's discussion about it. And disabling a lint rule for an individual file is allowed with proper reason documented along with it.

I will keep the current logic per above reason. Feel free to reach me if you have further ideas to discuss.